docs/getting-started/key-components.md

```diff
@@ -1,25 +1,45 @@
+A [connection](../reference/glossary.md#connection) establishes a link between nodes to route data through the workflow. Each node can have one or multiple connections.
+* *Example*: An [IF node](../nodes/nodes-library/core-nodes/If/README.md) has two connections to different nodes: one for the case when the statement is false and one for the case when the statement is true.
 
+![Connection](images/Connection.png)
 
 ## Node
 
-A node is an entry point for retrieving data, a function to process data or an exit for sending data. The data process includes filtering, recomposing and changing data. There can be one or several nodes for your API, service or app. You can connect multiple nodes, which allows you to create simple and complex workflows with them intuitively.
+A [node](../reference/glossary.md#node) is an entry point for retrieving data, a function to process data, or an exit for sending data. The data process includes filtering, recomposing, and changing data. There can be one or several nodes for your API, service, or app. By connecting multiple nodes, you can create simple and complex workflows. Nodes are named like the application they represent and are listed under Regular Nodes in the Editor.
+
+![Regular_nodes](images/Regular_nodes.png)
 
-For example, consider a Google Sheets node. It can be used to retrieve or write data to a Google Sheet.
+* *Example*: A [Google Sheets node](../nodes/nodes-library/nodes/GoogleSheets/README.md) can be used to retrieve or write data to a Google Sheet.
+
+![Sheets_node](images/Google_sheets.png)
+
+Note that if an application you need does not have a dedicated Node yet, you can access the data by using the [HTTP Request node](../nodes/nodes-library/core-nodes/HTTPRequest/README.md) or the [Webhook node](../nodes/nodes-library/core-nodes/Webhook/README.md). These nodes offer generic API support and are very versatile.
 
 
 ## Trigger Node
 
-A trigger node is a node that starts a workflow and supplies the initial data. What triggers it, depends on the node. It could be the time, a webhook call or an event from an external service.
+A [trigger node](../reference/glossary.md#trigger) is a node that starts a workflow and supplies the initial data. What triggers it, depends on the node. It could be the time, a webhook call, or an event from an external service. Trigger nodes are named like the application they represent followed by "Trigger" and are listed under Trigger Nodes in the Editor. 
 
-For example, consider a Trello trigger node. When a Trello Board gets updated, it will trigger a workflow to start using the data from the updated board.
+![Trigger_nodes](images/Trigger_nodes.png)
 
+* *Example*: A [Trello trigger node](../nodes/nodes-library/nodes/Trello/README.md) can be used to trigger a workflow when a Trello Board gets updated, using the data from the updated board.
```

Comment:
This creates an unordered list with a single item. We prefer to add lists if there is a list of items and avoid single item lists.